canonical-web-and-design/jaas-dashboard
# Add Eslint

## Done

Add Eslint

## QA

- Pull code
- Run `./run clean && ./run serve`
- Run `./run exec yarn run eslint-check`
- Verify no errors
- Check Eslint CI job passes below

## Details

Fixes #26
